pkg/pss/pss_test.go

```diff
@@ -223,8 +225,9 @@ func TestDeliver(t *testing.T) {
 
 	// create and register handler
 	var tt trojan.Topic // test variable to check handler func was correctly called
-	hndlr := func(m trojan.Message) {
+	hndlr := func(ctx context.Context, m trojan.Message) error {
 		tt = m.Topic // copy the message topic to the test variable
+		return nil
 	}
 	pss.Register(topic, hndlr)
```

Comment:
also, it's not showing up in the diff here, but on line `235` the `Deliver` call is also not error checked

Reply:
it is checked....

Reply:
now it is. thanks